mautic/mautic
# Company form fields

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds the ability to add company fields to form fields when building a form.
[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply PR, create a new form and select company fields to create the form. These fields should be associated with a company.
2. A company will only be created if the company name value is present.
3. A company will only be updated if it can be identified by: company name and (country, state, city)
